assets/tests/components/map.test.tsx

```diff
@@ -116,13 +116,16 @@ describe("map", () => {
   })
 
   test("performs onPrimaryVehicleSelected function when primary vehicle selected", async () => {
+    jest.spyOn(global, "scrollTo").mockImplementationOnce(jest.fn())
+
     const onClick = jest.fn()
     render(<Map vehicles={[vehicle]} onPrimaryVehicleSelect={onClick} />)
     await userEvent.click(screen.getByText(runIdToLabel(vehicle.runId!)))
     expect(onClick).toHaveBeenCalledWith(expect.objectContaining(vehicle))
   })
 
   test("does not perform onPrimaryVehicleSelected function when secondary vehicle selected", async () => {
+    jest.spyOn(global, "scrollTo").mockImplementationOnce(jest.fn())
```

Comment:
question(non-blocking): would you mind explaining why these were added? 
Seems like it's nessicary, but I don't have a good sense of why.

Reply:
These resolve console logs that were introduced in #1802 - see previous run [here](https://github.com/mbta/skate/actions/runs/3577274849/jobs/6016096725#step:8:33).

Clicking a leaflet marker calls `scrollTo` under the hood, which is why this is only added to tests with marker click interactions.

Reply:
I think this is to address some error output from running the tests that I noticed were introduced in #1802.